ui/src/shared/components/InputClickToEdit.js

```diff
@@ -63,7 +71,7 @@ class InputClickToEdit extends Component {
                 className="form-control input-sm provider--input"
                 defaultValue={value}
                 onBlur={this.handleInputBlur}
-                onKeyDown={this.handleKeyDown}
+                onKeyUp={this.handleKeyUp}
                 autoFocus={true}
                 onFocus={this.handleFocus}
                 ref={r => (this.inputRef = r)}
```

Comment:
It doesn't look like you're using inputRef anymore?

Reply:
if tabIndex is not specified in props, then does the tabIndex of the input become undefined? could it default to 0? so the input would behave as expected? https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex